### PR TITLE
Add collapse and fullscreen buttons to preview pane

### PR DESCRIPTION
**Pull request by maddiedierker**
Adds buttons and component state for "collapse" and "fullscreen" functionality in the Preview pane:

`<DefaultVisualization/>`:
Screen Shot 2021-05-26 at 12 08 41 PM

`NeighborhoodVisualizationColumn/>` (there was previously no pane header):
Screen Shot 2021-06-03 at 2 53 16 PM

Other things to note:
- This abstracts the preview pane header into a `<PreviewPaneHeader/>` component.
- Fixes a bug where the preview pane was displayed in "console" mode and when the mode is empty (project/standalone mode).
- The "fullscreen" button is hidden right now since we won't be implementing fullscreen for a while, but there is a placeholder in its place to keep everything centered in the header.

## Links

- [CSA-361](https://codedotorg.atlassian.net/browse/CSA-361)

### CODE REVIEW

File: apps/src/javalab/DefaultVisualization.jsx [hunk 1]
```diff
@@ -1,14 +1,53 @@
...
+          onClick={() => {}}
+          label=""
+          isRtl={false}
+          style={styles.transparent}
+          leftJustified
+        />
+        <PaneButton
+          headerHasFocus
+          iconClass={this.fullscreenIcon()}
+          onClick={() => {}}
```
Comment (molly-moen) [inline apps/src/javalab/DefaultVisualization.jsx]: My understanding is preview fullscreen won't be enabled until post-pilot. Do we still want this button here if it won't do anything for a long time? @jmkulwik is my understanding correct?

Comment (jmkulwik) [reply apps/src/javalab/DefaultVisualization.jsx]: That is true. Let me check in with @moneppo

Comment (maddiedierker) [reply apps/src/javalab/DefaultVisualization.jsx]: we're going to hide the fullscreen icon for now ([slack thread](https://codedotorg.slack.com/archives/C01EF4GJ9GE/p1622142834164000)). removing this element causes the "Preview" text to not be centered in the header, so i added an empty placeholder icon to keep the text centered. when we're ready to implement fullscreen, we can add `iconClass` back and remove `icon` here

**Comment by maddiedierker** [comment]
@code-dot-org/csa PTAL!

**Comment by maddiedierker** [comment]
nevermind, hold off on reviewing -- i'm going to refactor this PR to add these buttons to the neighborhood visualization as well

**Comment by maddiedierker** [comment]
@code-dot-org/csa this is ready for review now!

**Review by molly-moen — APPROVED** [review]
🎉

**Comment by maddiedierker** [comment]
@bencodeorg heads up that this PR merged, so you can add the "asset manager" button to the preview pane!

**Pull request merged**